### PR TITLE
New labels directory, fixes #57

### LINKED ISSUE #57: Need a better organization for label files
Need an organized directory for the labels files, including a naming convention for the individual labelers.

### PR DESCRIPTION
**Pull request by BoltzmannBrain**
Includes:
  - New labels directory, w/ subdirectory for individual label files
  - New label file naming convention: "XY_labels_v2.json" is version 2 of hand labels by person with initials XY
  - New combined labels file for most recent versions of Subutai's, Celeste's, and my own hand labels: "combined_labels.json"
    NOTE: nab is not yet updated to use this file and will still look for "ground_truth_labels.json"
  - Revised formatting and commenting of combine_labels.py and labeler.py
@subutai please review
@scottpurdy

### CODE REVIEW

File: nab/labeler.py [hunk 1]
```diff
@@ -99,14 +99,17 @@ def found(t, data):
...
-      num_windows = len(self.windows[relativePath])
-      if num_windows > 1:
-        if not all([(self.windows[relativePath][i+1][0]
-                    - self.windows[relativePath][i][1]).total_seconds() >= 0
-                    for i in range(num_windows-1)]):
-          raise ValueError("In the label file %s, windows overlap." % self.path)
+      # Check that windows are distinct (unique, non-overlapping); the end time
+      # of a window can be the same as the start time of the subsequent window.
+      # This check is not for the combined labels file, and the self.path
+      # condition must match the "--destPath" argument in combine_labels.py.
```
Comment (subutai) [inline nab/labeler.py]: Does this mean that the relaxed windows can overlap?

**Comment by subutai** [comment]
@BoltzmannBrain Looks good. I left one small question. In addition, I don't think we can consider #57 to be complete until we delete `ground_truth_labels.json` and switch default to this one. Is there a reason not to delete the old file?

File: nab/labeler.py (hunk 1 above)
Comment (BoltzmannBrain) [inline nab/labeler.py]: With the relaxed window margin set as is -- 10% of the dataset, no matter the number of total windows in the dataset -- the relaxed windows overlap, e.g.

"realAWSCloudwatch/ec2_cpu_utilization_825cc2.csv": [
        [
            "2014-04-14 22:59:00.000000",
            "2014-04-16 09:54:00.000000"
        ],
        [
            "2014-04-15 10:49:00.000000",
            "2014-04-16 21:09:00.000000"
        ]
    ]

This ValueError was written to catch these overlaps. After we discuss and modify the relaxed window parameter(s), the next commit will let it run normally :)

Comment (BoltzmannBrain) [inline nab/labeler.py]: I've revised the relaxed window calc to run as we discussed -- divide the length by the number of anomalies -- but there are still anomaly windows that overlap. Before the changes in this PR can be merged I think we need to revise the hand labels per our meeting today, and I'll find other ways to implement the relaxed window calculations.

Comment (BoltzmannBrain) [inline nab/labeler.py]: Merging this and will the relaxed window overlap issue (#58) with a different PR.

**Pull request merged**